GameArena.java

```diff
@@ -451,13 +452,23 @@ public void removeText(Text t)
 	}
 
 	/**
-	 * Pause for a 1/50 of a second. 
+	 * 
+	 * Sets the millis that the default {@link GameArena#pause()} method sleeps the {@link Thread} for
+	 * 
+	 * @param sleepMillis The time in milli seconds for which the {@link Thread} will sleep
+	 */
+	public void setSleepMillis(int sleepMillis) {
+		this.sleepMillis = sleepMillis;
+	}
+
+	/**
+	 * Pause for the configured milliseconds using {@link GameArena#setSleepMillis(int)}. 
 	 * This method causes your program to delay for 1/50th of a second. You'll find this useful if you're trying to animate your application.
 	 *
 	 */
 	public void pause()
 	{
-		this.pause(20);
+		this.pause(this.sleepMillis);
 	}
```

Comment:
Ta. Naming is not very intuitive though. Kinda want to go with 

```
setFrameRate(int milliseconds)
```

but that also implies we shoukd be using frequency and not period.

setFramePeriod() ? set frameDelay? setPauseDelay()?

Also, a matching accessor please, and make pause(ms) private?

Reply:
Alright I'm not sure what the most intuitive name would be. Probably setFrameDelay or setPauseDelay as that makes the most sense given what it doesn.

But doesn't making pause(ms) private also defeat the point? Perhaps someone wants to pause the thread for an atypical amount of time?

Reply:
also see line 143. We should probably set that to be half of the inter frame spacing in use.

Reply:
Let's go with setPauseDelay.

I think pause(ms) being private is the way to go, just for simplicity. Then it encourages users to instead use setPauseDelay() and pause(), which will result in simpler code for users in the general case.

I'm struggling to think of a reason why using a pause() that's not related to your frame rate, and also it uses method overloading (and i'm trying to keep the API consistent with topcis we've covered in the course)

Reply:
Alright, that sounds fairly logical to me. I can't see any reason for anyone to use pause(ms) either and if they're desperate I suppose they can always either create a new fork or use some sort of hacky reflection.

Reply:
yep - or use another timer. :)

p.s. could you replace millis with milliseconds please - SI units rule.